www/reviews.php

```diff
@@ -450,7 +456,10 @@ function showReview($db, $gameid, $rec, $specialNames, $optionFlags = 0)
             . "return false;\">Yes</a> &nbsp; "
             . "<a href=\"needjs\""
             . "onclick=\"javascript:sendReviewVote('$reviewid', 'N');"
-            . "return false;\">No</a> &nbsp; ";
+            . "return false;\">No</a> &nbsp; "
+            . "<a href=\"needjs\""
+            . "onclick=\"javascript:sendReviewVote('$reviewid', 'R');"
+            . "return false;\">Remove vote</a> &nbsp; ";
```

Comment:
I just noticed that it _always_ includes the "Remove vote" link, even if you've never voted, and even right after you've removed your vote. (But fixing that sounds like a hassle.)

WDYT? Do you want to take a crack at making the link only show up when it's needed? Or I can just merge this now and we can file that as a separate bug?